cmake/modules/BuildPlugin.cmake

```diff
@@ -60,7 +60,7 @@ MACRO(BUILD_PLUGIN PLUGIN_NAME)
 
 	IF(LMMS_BUILD_APPLE)
 		IF ("${PLUGIN_LINK}" STREQUAL "SHARED")
-			SET_TARGET_PROPERTIES(${PLUGIN_NAME} PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
+			TARGET_LINK_OPTIONS(${PLUGIN_NAME} PUBLIC "-undefined dynamic_lookup")
```

Comment:
@tresf this line fails on CI on macos. Can you help here?

Reply:
Sorry, linking is an area that I'm not very comfortable with.  Oddly, the error that the CI generates suggests you're close:

```
ld: invalid option to -undefined [ warning | error | suppress | dynamic_lookup ]
```

... according to GitHub, removing the double-quotes might do the trick: https://github.com/search?q=target_link_options++dynamic_lookup+language%3ACMake&type=code

Reply:
Thanks, this fixed it.